apps/ccf-eui/src/app/modules/results-browser/tissue-section-vis/tissue-section-vis.component.scss

```diff
@@ -13,11 +13,14 @@
     display: flex;
     position: relative;
     justify-content: space-evenly;
+    border-left: 1px solid #cccccc;
```

Comment:
Could you please link where in figma this comes from? These colors doesn't seem to come from the design system so I'm wondering why?!

Reply:
These are the colors in the original version of the EUI, they were copied here: https://www.figma.com/design/AxRgTJl5PUE7tNqk0VsMc3/Exploration-User-Interface-(EUI)?node-id=2833-34548&t=SrULEoXLtfXjsuPv-4

This probably needs to be specced out further.